scripts/build

```diff
@@ -51,8 +51,10 @@ fi
 cd "${ROOT}"
 if [[ "${static}" == "true" ]]; then
 	CGO_ENABLED=0 go build -installsuffix cgo -a -ldflags "${LDFLAGS} -s" -o $build_exe ./agent/
```

Comment:
Make indentation spacing consistent in this if-block, and also my preference would be to leave the 'else' as it is and add a if-block for when TARGET_OS == "windows", ie:

```
if [[ "${TARGET_OS}" == "windows" ]]; then
...
elif [[ "${static}" == "true" ]]; then
...
else
...
fi
```

Reply:
I have made the requested changes.